packages/playground/website/playwright/e2e/website-ui.spec.ts

```diff
@@ -0,0 +1,175 @@
+import { test, expect } from '../playground-fixtures.ts';
+import {
+	clickCreateInNewSiteModal,
+	clickSaveInEditSettings,
+	expandSiteView,
+	getSiteInfoRowValue,
+	getSiteTitle,
+	hasNetworkingEnabled,
+	openEditSettings,
+	openNewSiteModal,
+	selectPHPVersion,
+	selectWordPressVersion,
+	setNetworkingEnabled,
+} from './website-helpers.ts';
+import { Blueprint } from '@wp-playground/blueprints';
+
+// We can't import the SupportedPHPVersions versions directly from the remote package
+// because of ESModules vs CommonJS incompatibilities. Let's just import the
+// JSON file directly. @ts-ignore
+// eslint-disable-next-line @nx/enforce-module-boundaries
+import { SupportedPHPVersions } from '../../../../php-wasm/universal/src/lib/supported-php-versions.ts';
+// eslint-disable-next-line @nx/enforce-module-boundaries
+import * as MinifiedWordPressVersions from '../../../wordpress-builds/src/wordpress/wp-versions.json';
+
+test('should reflect the URL update from the navigation bar in the WordPress site', async ({
+	page,
+}) => {
+	await page.goto('./?url=/wp-admin');
+
+	await expandSiteView(page);
+
+	const inputElement = page.locator('input[value="/wp-admin/"]');
+	await expect(inputElement).toBeVisible();
+	await expect(inputElement).toHaveValue('/wp-admin/');
+});
+
+test('should switch between sites', async ({ page }) => {
+	await page.goto('./');
+
+	await openNewSiteModal(page);
+
+	const newSiteName = await page
+		.locator('input[placeholder="Playground name"]')
+		.inputValue();
+
+	await clickCreateInNewSiteModal(page);
+
+	await expect(await getSiteTitle(page)).toMatch(newSiteName);
+
+	const sidebarItem = page.locator(
+		'.components-button[class*="_sidebar-item"]:not([class*="_sidebar-item--selected_"])'
+	);
+	const siteName = await sidebarItem
+		.locator('.components-flex-item[class*="_sidebar-item--site-name"]')
+		.innerText();
+	await sidebarItem.isVisible();
```

Comment:
`Locator.isVisible` doesn't wait for the element to become visible — it only returns its current visibility. For asserting explicit visibility the `await expect( locator ).toBeVisible()` assertion should be used.

Reply:
I removed all `isVisible`